src/common/components/DataView.tsx

```diff
@@ -0,0 +1,60 @@
+import { DataObject } from '../types/NarrativeDoc';
```

Comment:
Would recommend moving this file and other DataView files into a DataView folder (like NarrativeList)